boilerplate/app/app.tsx

```diff
@@ -73,6 +73,7 @@ function App(props: AppProps) {
 
   const [areFontsLoaded] = useFonts(customFontsToLoad)
 
+  // @mst replace-next-line React.useEffect(() => {
```

Comment:
nit: something got a little funky here, once this run I see the two spaces on a separate line from the replaced line here:

<img width="561" alt="Screenshot 2024-07-30 at 2 36 12 PM" src="https://github.com/user-attachments/assets/80c19331-8c01-462e-aa9f-8b6c781ea92d">

There might be a way to fix this, I don't know this markup system yet, but I was also going to suggest we also acll prettier at the end of `remove-mst` like we do for `remove-mst-markup`

Reply:
good call! I noticed a couple of these spots, so I added a prettier step 👍